core/messaging/src/index.ts

```diff
@@ -0,0 +1,40 @@
+import { natsServiceFactory } from './providers/nats';
+import { WhisperService } from './providers/whisper';
+
+export const messagePayloadTypeJson = 'json';
+export const messagePayloadTypeBinary = 'binary';
+
+export const messagingProviderNats = 'nats';
+export const messagingProviderWhisper = 'whisper';
+
+export interface IMessagingService {
```

Comment:
Given there's a clean interface here I guess an additional implementation could easily be added.